File: Java/Basic Maths/ReverseNumber.java
public class ReverseNumber {
    
    public static int reverseNumber(int n) {
        int ans = 0;
        while(n != 0){
            int digit = n % 10;
            ans = ans * 10 + digit;
            n = n / 10;
        }
        return ans;
    }
    public static void main(String[] args) {
        
        int n = 123;
        int ans = reverseNumber(n);
        System.out.println(ans);
    }
}

File: JavaScript/001 Begineer Problems/004 Concept Basics/Basic Maths/reverseNumber.js
function reverseNumber(n) {
    let ans = 0;
    while(n != 0){
        let digit = n % 10;
        ans = ans * 10 + digit;
        n = Math.floor(n / 10);
    }
    return ans;
}
let result = reverseNumber(123);
console.log(result)

// With String
function reverseNumber2(n){
    return Math.abs(n).toString().split('').reverse().join('');
}

let result2 = reverseNumber2(456);
console.log(result2);